### PR TITLE
Authentication step 3

### PR DESCRIPTION
**Pull request by Obinnabii**
final step in the authentication tutorial.
- add user emails when comments are posted and retrieved
- show email of the comment posters

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/data/Comment.java [hunk 1]
```diff
@@ -17,16 +17,20 @@
 public final class Comment {
 
   private final long id;
-  private final String text;
   private final long postTime;
+  private final String text;
+  private final String  userEmail;
```
Comment (Richie78321) [inline portfolio/src/main/java/com/google/sps/data/Comment.java]:
```suggestion
  private final String userEmail;
```
nit: remove extra space

File: portfolio/src/main/java/com/google/sps/servlets/PostCommentServlet.java [hunk 2]
```diff
@@ -39,12 +41,25 @@
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String text = request.getParameter("comment-input").trim();
     if (!text.isBlank()) { 
-      long commentPostTime = System.currentTimeMillis();
       Entity commentEntity = new Entity(Comment.ENTITY_NAME);
       commentEntity.setProperty(Comment.TEXT_FIELD, text);
+      long commentPostTime = System.currentTimeMillis();
       commentEntity.setProperty(Comment.POST_TIME_FIELD, commentPostTime);
+      String commentUserEmail = getUserEmail();
+      commentEntity.setProperty(Comment.USER_EMAIL_FIELD, commentUserEmail);
```
Comment (Richie78321) [inline portfolio/src/main/java/com/google/sps/servlets/PostCommentServlet.java]:
```suggestion
      commentEntity.setProperty(Comment.USER_EMAIL_FIELD, getUserEmail());
```
nit: could be reduced to one line without sacrificing much readability

File: portfolio/src/main/webapp/comments.css [hunk 3]
```diff
@@ -89,3 +89,12 @@ small {
 #auth-button.logout:hover {
   background-color: rgb(146, 7, 0);
 }
+
+.user-email {
+  font-weight: bold;
+  color: rgb(0, 130, 0)
```
Comment (Richie78321) [inline portfolio/src/main/webapp/comments.css]:
```suggestion
  color: rgb(0, 130, 0);
  font-weight: bold;
```
nit: alphabetize declarations, also missing semicolon
https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order

File: portfolio/src/main/java/com/google/sps/servlets/PostCommentServlet.java (hunk 2 above)
Comment (Obinnabii) [reply portfolio/src/main/java/com/google/sps/servlets/PostCommentServlet.java]: yeah, that makes a lot of sense

File: portfolio/src/main/java/com/google/sps/data/Comment.java (hunk 1 above)
Comment (Obinnabii) [reply portfolio/src/main/java/com/google/sps/data/Comment.java]: great catch

**Review by billyeh — APPROVED** [review]

**Comment by billyeh** [comment]
Great job!

**Pull request merged**